code/src/Input.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './Input.css'
+
+
+export const Input = (props) => {
+
+    return (
+        <label className="input">
```

Comment:
It can be confusing with a className called input for a label. Maybe you can change it to `inputTitle` or emailLabel, so it is not confused with the actual input?